src/adapters/desyn/execution/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import { UserTVLData } from './sdk/types';
+import { getUserTVLData } from './sdk/lib';
+
+
+export const getUserTransactionData = async (blockNumber: number): Promise<UserTVLData[]> => {
+  return await getUserTVLData(blockNumber);
+};
+
```

Comment:
it should export a function named `getUserTVLData`

Reply:
I resubmitted it.  export  function getUserTVLData